Description: Correctly cleanup grabbing accelerators
 Test patch to examine if can resolve occasional shortcuts
 stopping working. Will be forwarded upstream after
 further community testing
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: No
Last-Update: 2019-03-09

Index: budgie-desktop-10.5~git20190308/src/wm/shim.vala
===================================================================
--- budgie-desktop-10.5~git20190308.orig/src/wm/shim.vala
+++ budgie-desktop-10.5~git20190308/src/wm/shim.vala
@@ -222,7 +222,7 @@ public class ShellShim : GLib.Object
                 continue;
             }
             display.ungrab_accelerator(key);
-            grabs.remove(key);
+            iter.remove();
         }
     }

